Resources/Prototypes/_DV/Entities/Mobs/Species/chitinid.yml

```
@@ -135,6 +135,10 @@
     understands:
     - TauCetiBasic
     - Moffic
+  - type: ItemCougher #Durk fully implements a bounty challenge ( IMPOSSIBLE )
```

Comments are funny, but unnecessary.
```suggestion
  - type: ItemCougher
```